src/Claim/LesserOrEqualsTo.php

```diff
@@ -23,12 +24,18 @@ class LesserOrEqualsTo extends Basic implements Claim, Validatable
     /**
      * {@inheritdoc}
      */
-    public function validate(ValidationData $data): bool
+    public function validate(ValidationData $data)
     {
-        if ($data->has($this->getName())) {
-            return $this->getValue() <= $data->get($this->getName());
+        $name = $this->getName();
+        if ($data->has($name) && ($this->getValue() > $data->get($name))) {
+            throw new InvalidClaimException(
+                sprintf(
```

Comment:
You're repeating yourself with `sprintf($message, $name, $data->get($name), $this->getValue())`